bitrise.yml

```diff
@@ -755,10 +1136,22 @@ app:
   - opts:
       is_expand: false
     BITRISE_EXPORT_METHOD: development
+  - opts:
+      is_expand: false
+    BITRISE_NIGHTLY_VERSION: '40'
+  - opts:
+      is_expand: false
+    BITRISE_RELEASE_VERSION: '95'
+  - opts:
+      is_expand: false
+    BITRISE_BETA_VERSION: '95'
 trigger_map:
 - push_branch: main
   workflow: RunUnitTests
 - push_branch: v39.0
-  workflow: xcode13-release-and-beta-nocache
+  workflow: Deploy_Prod_Beta
 - pull_request_target_branch: main
   workflow: RunUnitTests
+meta:
```

Comment:
Not sure this is needed as this is setting a general stack while we have each stack defined per workflow which is used insteao of this one.